CrossPare/src/main/java/de/ugoe/cs/cpdp/execution/WithinProjectOrderedSplitExperiment.java

```diff
@@ -164,7 +164,7 @@ public void run() {
                         }
                     }
                 }
-
+                SoftwareVersion trainversionOriginal = new SoftwareVersion(testversion);

```

Comment:
is testversion here correct? the code above splits the instances based on their index, the training data should be in the trainversion.